website/src/pages/api/user_stats.ts

```diff
@@ -3,7 +3,14 @@ import { createApiClientFromUser } from "src/lib/oasst_client_factory";
 import { getBackendUserCore } from "src/lib/users";
 
 const handler = withoutRole("banned", async (req, res, token) => {
-  const user = await getBackendUserCore(token.sub);
+  let userid: string;
+  if (req.query.uid) {
```

Comment:
Also can you check the current user has any mod or admin role herer? We dont want to expose stats to every one.

Reply:
done